### ISSUE TITLE
Term 1 - Lesson 20 - 27. Color Classify typo

### ISSUE THREAD

**Issue by earlbread** [udacity/sdc-issue-reports#570]
In the middle of the lesson,

> And now we're ready to shuffle and split the data into training and testing sets. **To to** this we'll use the Scikit-Learn train_test_split() function, but it's worth noting that recently, this function moved from the sklearn.cross_validation package (in sklearn version <=0.17) to the sklearn.model_selection package (in sklearn version >=0.18).

should be

> And now we're ready to shuffle and split the data into training and testing sets. **To do** this we'll use the Scikit-Learn train_test_split() function, but it's worth noting that recently, this function moved from the sklearn.cross_validation package (in sklearn version <=0.17) to the sklearn.model_selection package (in sklearn version >=0.18).

Thank you!

**Comment by swwelch** [comment]
Hi @earlbread, I'm having trouble tracking down the text above. Do you have a link for that concept?

**Comment by earlbread** [comment]
Hi @swwelch,

Sure, the link is [here](https://classroom.udacity.com/nanodegrees/nd013/parts/fbf77062-5703-404e-b60c-95b78b2f3f9e/modules/2b62a1c3-e151-4a0e-b6b6-e424fa46ceab/lessons/fd66c083-4ccb-4fe3-bda1-c29db76f50a0/concepts/be308636-742b-416a-8fcc-c6071865a11f).

I'm attaching the capture below.
![screen shot 2017-03-30 at 5 24 49 pm](https://cloud.githubusercontent.com/assets/5907919/24494683/d66e2d22-156d-11e7-8ef1-b1011f1a609b.jpg)

Thank you!

**Comment by swwelch** [comment]
Thanks! Fixed now.
